Add default locale config to the documentation (#3906)
wso2/docs-is: en/docs/learn/customizing-automated-emails.md

    
        For example, use **fr\_fr** for the French locale, **pt\_br** for Brazil locale and **it\_it** for the Italian locale. This is the format used for country locales in the Identity Server.
        For more information on country locales, see [Locale codes](http://www.science.co.il/language/Locale-codes.php).

    !!! note

>         If the local value is not defined for the user, the default locale value will be “en_US”. From the update level 250 you can change the default locale value using the following configuration,

Reviewer: ```suggestion
        If the locale value is not defined for the user, the default locale value will be `en_US`. From update level **5.11.0.250**, you can change the default locale value using the following configuration:
```

        ``` 
        [identity_mgt.notification]
        default_locale = "<locale>"
        ```

>         If the above configuration is defined in the deployment.toml file, it will be used as the locale for both SMS and email flows; otherwise, the default value (en_US) will be assigned.

Reviewer: ```suggestion
        If the above configuration is added to the `deployment.toml` file, the specified value will be used as the locale for both SMS and email flows; otherwise, the default value (`en_US`) will be assigned.
```

>         Please note that even after adding this configuration, if the registry doesn't have the template related to the configured locale, the configuration will not be honored, and the default locale will be set to en_US.

Reviewer: ```suggestion
        Note that if the registry doesn't have the template related to the newly configured locale, the configuration will not be considered, and the configurations will be reverted to the default locale.
```
